### DISCUSSION TITLE
Visible grid?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#3928] [Q&A]
Is there currently any facility for displaying the snapping grid in a visible form? Or indeed any other type of visible grid? I can find noting like this in preferences, but I've missed stuff before so ...

**Comment by robinp7720** [comment]
The snapping grid is aligned by default to the grid lines in the graph background

**Reply by NelkonParker** [reply]
So it's not possible to display it when editing an arbitrary pdf, I guess? If so, I will raise an enhancement request.
